src/dotnet/Microsoft.DotNet.Core.Sdk.props

```diff
@@ -15,7 +15,8 @@
 
   <!-- User-facing configuration-specific defaults -->
   <PropertyGroup>
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <!-- TODO: This property can't be set in the CLI since ResolveNuGetPackageAssets needs an exact RID from the project.json, and the RID is made up of the PlatformTarget -->
+    <!--<PlatformTarget>AnyCPU</PlatformTarget>-->
```

Comment:
Not sure what the right fix here is, perhaps something is smart enough to infer x86 and x64? Or we add support for non-concrete rids?